package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "openLRSng-configurator",
+  "description": "openLRSng Configurator",
+  "version": "3.8.8",
+  "main": "main.html",
+  "default_locale": "en",
+  "scripts": {
+    "start": "node_modules/gulp/bin/gulp.js build && node_modules/nw/bin/nw ."
+  },
+  "window": {
+    "title": "openLRSng Configurator",
+    "icon": "images/icon_128.png",
+    "toolbar": true,
+    "width": 1024,
+    "height": 800
+  },
+  "repository": {
+    "type": "git",
+    "url": "github.com/openLRSng/openLRSng-configurator"
+  },
+  "author": "openLRSng",
+  "license": "GPL-3.0",
+  "dependencies": {
```

Comment:
That's an awful lot of runtime dependencies. Are you sure that all of these (including `gulp`, `nw-builder`) are required at runtime?

Reply:
Please, see my previous answer to your first reply.

Reply:
Take a look to iNav and betaflight package.json files:

[https://github.com/iNavFlight/inav-configurator/blob/master/package.json](https://github.com/iNavFlight/inav-configurator/blob/master/package.json)

[https://github.com/betaflight/betaflight-configurator/blob/master/package.json](https://github.com/betaflight/betaflight-configurator/blob/master/package.json)

Reply:
This still does not explain why your `package.json` is declaring so many runtime dependencies, with a lot of them looking like they should be dev dependencies.

Reply:
Oh! I apologyze. You can remove the dependencies and script sections, it is only needed to build from development version.

Finally the package.json would be like this:

{
  "name": "openLRSng-configurator",
  "description": "openLRSng Configurator",
  "version": "3.8.8",
  "main": "main.html",
  "default_locale": "en",
  "window": {
    "title": "openLRSng Configurator",
    "icon": "images/icon_128.png",
    "toolbar": true,
    "width": 1024,
    "height": 800
  },
  "repository": {
    "type": "git",
    "url": "github.com/openLRSng/openLRSng-configurator"
  },
  "author": "openLRSng",
  "license": "GPL-3.0"
}

Reply:
In this case you are not installing any dependencies. This will certainly not work, since things like `nw-builder` are required to **build** the standalone app - they just are not required to **run** the standalone app.

I suggest we close this, and wait until me (or somebody else) ports the build system that is used for the Betaflight configurator to the openLRSng configurator. It is on top of my priority list, and there are a couple of holidays coming up...